vendor/github.com/go-sql-driver/mysql/dsn.go

```diff
@@ -274,6 +274,10 @@ func (cfg *Config) FormatDSN() string {
 		writeDSNParam(&buf, &hasParam, "tls", url.QueryEscape(cfg.TLSConfig))
 	}
 
+	if cfg.TLS != nil {
```

Comment:
This will be overwritten during build in our `GNUmakefile`.

It cannot remain here.

Reply:
I'm not sure I follow. Why is it overwritten?

Reply:
It's a part of your PR - so I believe you changed it in one file and then pushed it.

As you are adding TLS config called "custom", maybe you'll have to set `cfg.TLSConfig` to `custom` in your code (see the condition just above yours).
It will be in your code, not in the library code.

Reply:
I see it now, I didn't realise initially that I was actually inside the vendor directly and thought it was being handled by the provider. I've changed the approach a bit to account for that

Reply:
Ok, could you please remove that vendor directory and run `make deps`? That should remove the remaining files you added, that are no longer needed.